AlphaWallet/InCoordinator.swift

```diff
@@ -420,7 +420,7 @@ class InCoordinator: Coordinator {
         let s = "0x" + signature.substring(with: Range(uncheckedBounds: (64, 128)))
         guard let wallet = keystore.recentlyUsedWallet else { return }
 
-        ClaimOrderCoordinator(web3: web3).claimOrder(
+        ClaimOrderCoordinator(web3: web3, config: self.config).claimOrder(
```

Comment:
I haven't look over the whole thing yet. But code following this line should be changed too:

```
        ClaimOrderCoordinator(web3: web3, config: self.config).claimOrder(
```

The old version of `claimOrder()` (despite the name) only encodes the function call and prepares the transaction payload and the payload is then used to populate a transaction in the completion block passed to `claimOrder()`, starting at line 434.

Since we are now encoding and then calling `callSmartContract()` (this is good!) in `claimOrder()`, the completion block passed to it should be much simplified.

In other words,

* Old version of `claimOrder()` only encodes the function call
* New version of `claimOrder()` encodes + makes the function call

Reply:
Sussed. Tests pass on your machine?